gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -2448,6 +2468,49 @@ properties:
         $ref:
           - "_terms_enum.yaml#/stage_ivb/diagnosis/masaoka_stage"
 
+  max_tumor_bulk_site:
+    $ref:
+      - "_terms.yaml#/max_tumor_bulk_site/common"
+    enum:
+      - Adrenal
+      - Appendix
+      - Ascites/Peritoneum
+      - Axillary lymph nodes
+      - Bone Marrow
+      - Brain
+      - Breast
+      - Cervical lymph nodes
+      - Colon
+      - Iliac
+      - Iliac-external
+      - Inguinal
+      - Kidney
+      - Liver
+      - Lung
+      - Mandible
+      - Maxilla
+      - Mediastinal lymph nodes
+      - Mediastinal Soft Tissue
+      - Mesenteric lymph nodes
+      - Neck
+      - No Known Nodal Involvement
+      - Oropharynx
+      - Other
+      - Ovary
+      - Pancreas
+      - Paraaortic lymph nodes
+      - Parotid Gland
+      - Pericardium
+      - Peri-orbital Soft Tissue
+      - Retroperitoneal lymph nodes
+      - Small Intestine
+      - Soft Tissue (muscle, ligaments, subcutaneous)
+      - Splenic lymph nodes
+      - Stomach
+      - Submandibular lymph nodes
+      - Supraclavicular lymph nodes
+      - Thyroid
+
```

Comment:
Will need to change to consistent case.  I suggest sentence-case.

Reply:
not spotting any non-sentence-case items here left.